samples/subsys/zbus/msg_subscriber/src/main.c

```diff
@@ -183,7 +199,7 @@ int main(void)
 
 	while (1) {
 		LOG_INF("----> Publishing to %s channel", zbus_chan_name(&acc_data_chan));
-		zbus_chan_pub(&acc_data_chan, &acc, K_SECONDS(1));
+		zbus_chan_pub(&acc_data_chan, &acc, K_NO_WAIT);
```

Comment:
I'm also curious whether this is a non-related change

Reply:
Removed.